src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -191,17 +192,43 @@ class PairedResult extends GraphContent {
      */
     resize(graph) {
         if (utils.notEmpty(this.dataTarget.regions)) {
-            // If graph has more than 1 content then we dont load the regions
+            // If graph has more than 1 content, we compare the regions if they are identical and hide if even atleast one of them is not.
+            const regionList = this.dataTarget.regions;
+            const values = this.dataTarget.values;
+            //check if all region are there with respect to value (high, mid and low)
             if (graph.content.length > 1) {
-                hideAllRegions(graph.svg);
+                const isRegionThere = (value) => {
+                    for (const key in value) {
+                        if (!regionList.hasOwnProperty(key)) {
+                            return false;
+                        }
+                    }
+                    return true;
+                };
+                const isPairedDataProper = values.every(isRegionThere);
+
+                if (
+                    isPairedDataProper === true &&
```

Comment:
```js
if (
    isPairedDataProper &&
    !graph.config.shouldHideAllRegion &&
    areRegionsIdentical(graph.svg)
)
```

Reply:
yes i will make the change

Reply:
You can shorten this to `isPairedDataProper` instead of `isPairedDataProper === true`.

Reply:
made the change.